### ISSUE TITLE
NTR: alpha-N-acetylglucosaminidase low in leukocytes or cultured fibroblasts (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7933]
New term request alpha-N-acetylglucosaminidase low in leukocytes or cultured fibroblasts (f382): N-acetyl-alpha-D-glucosaminidase

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000360
- label: Reduced tissue alpha-N-acetylglucosaminidase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:12202988
- def: Concentration or activity of alpha-N-acetylglucosaminidase (EC 3.2.1.50) below the lower limit of normal. This enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts. It is also known as alpha-acetylglucosaminidase, N-acetyl-alpha-D-glucosaminidase, N-acetyl-alpha-glucosaminidase, and alpha-D-2-acetamido-2-deoxyglucosidase.
- comment: Pathogenic variants in the gene encoding N-alpha-acetylglucosaminidase (NAGLU) cause Sanfilippo syndrome B (mucopolysaccharidosis type IIIB).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7933
- synonyms:
